### DISCUSSION TITLE
Cannot read properties of undefined when I am uploading file

### DISCUSSION THREAD

**Discussion by PhatNguyenAI** [danny-avila/LibreChat#3547] [Troubleshooting]
### What happened?

I have a bug when I am upload file, content bug below:
2024-08-05 17:04:46 error: Error embedding file Cannot read properties of undefined (reading 'status')
2024-08-05 17:04:46 error: [/files] Error processing file: Cannot read properties of undefined (reading 'status')
Please, you help me..! thank

### Steps to Reproduce

<img width="1126" alt="image" src="https://github.com/user-attachments/assets/0dc10832-73ca-4afb-b1c7-a240aeac24f9">

### Screenshots

<img width="803" alt="Screenshot 2024-08-05 171021" src="https://github.com/user-attachments/assets/4e3d4303-e791-42a5-9707-15993aa430df">

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
it's clear from your screenshot, "RAG API is either not running or not reachable..."

https://www.librechat.ai/docs/configuration/rag_api
